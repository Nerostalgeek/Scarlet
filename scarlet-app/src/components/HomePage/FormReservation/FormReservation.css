.reservation-box {
  background: white;
  width: 50%;
  padding: 50px;
  border-radius: 10px;
  text-align: center;
  margin: auto;
  -webkit-box-shadow: 0px 5px 20px 0px rgba(0, 0, 0, 0.75);
  -moz-box-shadow: 0px 5px 20px 0px rgba(0, 0, 0, 0.75);
  box-shadow: 0px 5px 20px 0px rgba(0, 0, 0, 0.75);
}

#reservation-box-title {
  font-size: 30px;
  padding-bottom: 25px;
}

.form-reservation-row {
  margin-bottom: 15px;
  display: flex;
  justify-content: center;
  align-items: center;
  text-align: center;
}

.reservation-row-item {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  width: 100%;
}

.form-reservation input {
  border: solid 1px #c0c0c0;
  font-size: 18px;
  padding: 10px;
  width: 100%;
  border-radius: 10px;
}

.reservation-label {
  text-align: center;
  padding-bottom: 5px;
}

.submitInput {
  cursor: pointer;
  border: none;
  transition: 0.2s ease;
}

.input-disabled {
  opacity: 0;
  pointer-events: none;
  background: gray;
  cursor: crosshair;
  width: 60%;
}

.input-enabled {
  opacity: 1;
  width: 80%;
  pointer-events: auto;
  background: gray;
  background: #00c6ff; /* fallback for old browsers */
  background: -webkit-linear-gradient(
    to right,
    #2c58c4,
    #3f8ffe
  ); /* Chrome 10-25, Safari 5.1-6 */
  background: linear-gradient(
    to right,
    #2c58c4,
    #3f8ffe
  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
  color: white;
  cursor: pointer;
}
