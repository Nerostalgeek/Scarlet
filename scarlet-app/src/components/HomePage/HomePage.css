.l-homepage {
  height: 100%;
  text-align: center;
  overflow: hidden;
  padding-top: 75px;
}

.homepage-landing {
  height: calc(100vh - 75px);
  background-image: url("../../img/assets/landing/background-homepage.jpg");
  background-size: cover;
  background-repeat: no-repeat;
  display: flex;
  justify-content: space-around;
  align-items: center;
  padding: 0 100px;
}

.homepage-landing-text {
  flex: 1;
  padding: 0 50px;
  text-align: left;
}

.homepage-reservation {
  flex: 1;
}

.homepage-landing-text #homepage-landing-title {
  color: white;
  font-size: 60px;
  padding-bottom: 25px;
  font-family: "Quicksand", sans-serif;
  line-height: 1.1;
}

.homepage-landing-text #homepage-landing-description {
  font-size: 20px;
  padding-bottom: 25px;
  font-family: "Open-Sans", sans-serif;
  color: white;
}

.landing-cta {
  background: #fc4a1a; /* fallback for old browsers */
  background: -webkit-linear-gradient(
    to top right,
    #fc4a1a,
    #ff5f38
  ); /* Chrome 10-25, Safari 5.1-6 */
  background: linear-gradient(
    to top right,
    #fc4a1a,
    #ff5f38
  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */

  color: white;
  border: none;
  margin-right: 10px;
  margin-bottom: 15px;
  padding: 15px 25px;
  font-size: 18px;
  border-radius: 5px;
  cursor: pointer;
}

.landing-cta:last-child {
  background: #00c6ff; /* fallback for old browsers */
  background: -webkit-linear-gradient(
    to right,
    #2c58c4,
    #3f8ffe
  ); /* Chrome 10-25, Safari 5.1-6 */
  background: linear-gradient(
    to right,
    #2c58c4,
    #3f8ffe
  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
}

.homepage-button {
  margin: 0 5px;
}

.homepage-reservation {
  width: 40%;
}
