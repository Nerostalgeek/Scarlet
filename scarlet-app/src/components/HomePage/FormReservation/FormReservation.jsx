import React, { useState, useEffect } from "react";
import Calendar from "react-calendar";
import Counter from "../../Counter/Counter";

import "./FormReservation.css";

const FormReservation = () => {
  const [citySelected, setCitySelected] = useState("");
  const [count, setCount] = useState(1);
  const [date, setDate] = useState(new Date());
  const [selectedRange, setSelectedRange] = useState("");

  const incrementCounter = event => {
    setCount(count + 1);
  }
  const decrementCounter = event => {
    if (count > 1) {
      setCount(count - 1);
    }
  }

  const onChangeCalendarValue = date => {
    let newRange = [...date];
    newRange = [
      newRange[0].toLocaleDateString(),
      newRange[1].toLocaleDateString()
    ];
    setSelectedRange(newRange);
  }

  return (
    <div className="reservation-box">
      <h3 id="reservation-box-title">
        Trouvez le véhicule parfait dès aujourd'hui
      </h3>

      <form className="form-reservation" action="">
        <div className="form-reservation-row">
          <div className="reservation-row-item">
            <p className="reservation-label">Ville</p>
            <input
              type="text"
              value={citySelected}
              onChange={event => setCitySelected(event.target.value)}
            />
          </div>
        </div>
        <p className="reservation-label">Durée du trajet</p>
        <div className="form-reservation-row">
          <Calendar
            returnValue={"range"}
            minDate={new Date()}
            onChange={onChangeCalendarValue}
            selectRange={true}
          />
        </div>

        <div className="form-reservation-row">
          <div className="form-reservation-counter">
            <p className="reservation-label">Nombre de passager(s)</p>
            <Counter
              initialCount={count}
              incrementCounter={incrementCounter}
              decrementCounter={decrementCounter}
            />
          </div>
        </div>
        <input id="submitInput" type="submit" value="Rechercher" />
      </form>
    </div>
  );
};

export default FormReservation;
