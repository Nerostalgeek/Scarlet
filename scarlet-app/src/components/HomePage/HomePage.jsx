import React from "react";

import landingIllustration from "../../img/assets/landing/landing-car-illustration.svg";
import "./HomePage.css";

import FormReservation from "./FormReservation/FormReservation";

const HomePage = () => {
  return (
    <div className="l-homepage">
      <div className="homepage-landing">
        <div className="homepage-landing-text">
          <h2 id="homepage-landing-title">
            Commencez à économiser grâce à Scarlet
          </h2>
          <p id="homepage-landing-description">
            Que vous souhaitez gagner de l'argent avec votre véhicule ou trouver
            un véhicule abordable pour votre week-end, vous êtes au bon endroit
            !
          </p>
          <button className="landing-cta">
            Explorer les voitures disponibles
          </button>
          <button className="landing-cta">Mettre ma voiture en location</button>
        </div>
        <div className="homepage-reservation">
          <FormReservation />
        </div>
      </div>
    </div>
  );
};

export default HomePage;
