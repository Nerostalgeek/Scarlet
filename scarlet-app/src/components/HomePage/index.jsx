import React from "react";
import API from "../../utils/API.js";

import "./HomePage.css";

import { Link } from "react-router-dom";

const HomePage = () => {
  return (
    <div className="l-homepage">
      <Link to="/login">
        <button className="homepage-button" type="primary">
          Se connecter
        </button>
      </Link>
    </div>
  );
};

export default HomePage;
