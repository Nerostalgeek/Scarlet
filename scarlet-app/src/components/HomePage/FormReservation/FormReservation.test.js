import React from "react";

import Adapter from "enzyme-adapter-react-16";
import { configure, mount } from "enzyme";

import renderer from "react-test-renderer";

import FormReservation from "./FormReservation";
import HomePage from "../HomePage";

configure({ adapter: new Adapter() });

describe("<FormReservation />", () => {
  test("Matches the snapshot", () => {
    const tree = renderer.create(<FormReservation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

it("Starts with a count of 0 for the number of passengers", () => {
  const wrapper = mount(<HomePage />);
  const text = wrapper.find(".counter-value").text();
  expect(text).toEqual("0");
});

it("Increments count by 1 on increment button click", () => {
  const wrapper = mount(<HomePage />);
  const incrementButton = wrapper.find(".increment-button");
  incrementButton.simulate("click");
  const text = wrapper.find(".counter-value").text();
  expect(text).toEqual("1");
});

it("Does not have a value under 1 on decrement click", () => {
  const wrapper = mount(<HomePage />);
  const incrementButton = wrapper.find(".increment-button");
  incrementButton.simulate("click");
  incrementButton.simulate("click");
  const decrementButton = wrapper.find(".decrement-button");
  decrementButton.simulate("click");
  decrementButton.simulate("click");
  decrementButton.simulate("click");
  decrementButton.simulate("click");
  const text = wrapper.find(".counter-value").text();
  expect(text).toEqual("1");
});

it('Submit button has class "input-disabled" if the fields are empty', () => {
  const wrapper = mount(<HomePage />);
  const submitButton = wrapper.find(".submitInput");
  expect(submitButton.prop("className")).toEqual("submitInput input-disabled");
});
