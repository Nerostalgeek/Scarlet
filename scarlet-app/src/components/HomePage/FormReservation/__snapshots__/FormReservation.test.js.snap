// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormReservation /> Matches the snapshot 1`] = `
<div
  className="reservation-box"
>
  <h3
    id="reservation-box-title"
  >
    Trouvez le véhicule parfait dès aujourd'hui
  </h3>
  <form
    action=""
    className="form-reservation"
    onSubmit={[Function]}
  >
    <div
      className="form-reservation-row"
    >
      <div
        className="reservation-row-item"
      >
        <p
          className="reservation-label"
        >
          Ville
        </p>
        <input
          id="selectCityInput"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <p
      className="reservation-label"
    >
      Durée du trajet
    </p>
    <div
      className="form-reservation-row"
    >
      <div
        className="react-calendar react-calendar--selectRange"
      >
        <div
          className="react-calendar__navigation"
          style={
            Object {
              "display": "flex",
            }
          }
        >
          <button
            aria-label=""
            className="react-calendar__navigation__arrow react-calendar__navigation__prev2-button"
            disabled={true}
            onClick={[Function]}
            type="button"
          >
            «
          </button>
          <button
            aria-label=""
            className="react-calendar__navigation__arrow react-calendar__navigation__prev-button"
            disabled={true}
            onClick={[Function]}
            type="button"
          >
            ‹
          </button>
          <button
            aria-label=""
            className="react-calendar__navigation__label"
            disabled={false}
            onClick={[Function]}
            style={
              Object {
                "flexGrow": 1,
              }
            }
            type="button"
          >
            January 2020
          </button>
          <button
            aria-label=""
            className="react-calendar__navigation__arrow react-calendar__navigation__next-button"
            onClick={[Function]}
            type="button"
          >
            ›
          </button>
          <button
            aria-label=""
            className="react-calendar__navigation__arrow react-calendar__navigation__next2-button"
            onClick={[Function]}
            type="button"
          >
            »
          </button>
        </div>
        <div
          className="react-calendar__viewContainer"
          onBlur={[Function]}
          onMouseLeave={[Function]}
        >
          <div
            className="react-calendar__month-view "
          >
            <div
              style={
                Object {
                  "alignItems": "flex-end",
                  "display": "flex",
                }
              }
            >
              <div
                style={
                  Object {
                    "flexGrow": 1,
                    "width": "100%",
                  }
                }
              >
                <div
                  className="react-calendar__month-view__weekdays"
                  onFocus={[Function]}
                  onMouseOver={[Function]}
                  style={
                    Object {
                      "display": "flex",
                      "flexDirection": undefined,
                      "flexWrap": "no-wrap",
                    }
                  }
                >
                  <div
                    className="react-calendar__month-view__weekdays__weekday"
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                  >
                    <abbr
                      aria-label="Monday"
                      title="Monday"
                    >
                      Mon
                    </abbr>
                  </div>
                  <div
                    className="react-calendar__month-view__weekdays__weekday"
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                  >
                    <abbr
                      aria-label="Tuesday"
                      title="Tuesday"
                    >
                      Tue
                    </abbr>
                  </div>
                  <div
                    className="react-calendar__month-view__weekdays__weekday"
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                  >
                    <abbr
                      aria-label="Wednesday"
                      title="Wednesday"
                    >
                      Wed
                    </abbr>
                  </div>
                  <div
                    className="react-calendar__month-view__weekdays__weekday"
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                  >
                    <abbr
                      aria-label="Thursday"
                      title="Thursday"
                    >
                      Thu
                    </abbr>
                  </div>
                  <div
                    className="react-calendar__month-view__weekdays__weekday"
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                  >
                    <abbr
                      aria-label="Friday"
                      title="Friday"
                    >
                      Fri
                    </abbr>
                  </div>
                  <div
                    className="react-calendar__month-view__weekdays__weekday"
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                  >
                    <abbr
                      aria-label="Saturday"
                      title="Saturday"
                    >
                      Sat
                    </abbr>
                  </div>
                  <div
                    className="react-calendar__month-view__weekdays__weekday"
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                  >
                    <abbr
                      aria-label="Sunday"
                      title="Sunday"
                    >
                      Sun
                    </abbr>
                  </div>
                </div>
                <div
                  className="react-calendar__month-view__days"
                  style={
                    Object {
                      "display": "flex",
                      "flexDirection": undefined,
                      "flexWrap": "wrap",
                    }
                  }
                >
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                    disabled={true}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="December 30, 2019"
                    >
                      30
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                    disabled={true}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="December 31, 2019"
                    >
                      31
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    disabled={true}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 1, 2020"
                    >
                      1
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    disabled={true}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 2, 2020"
                    >
                      2
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    disabled={true}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 3, 2020"
                    >
                      3
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__tile--now react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 4, 2020"
                    >
                      4
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 5, 2020"
                    >
                      5
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 6, 2020"
                    >
                      6
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 7, 2020"
                    >
                      7
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 8, 2020"
                    >
                      8
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 9, 2020"
                    >
                      9
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 10, 2020"
                    >
                      10
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 11, 2020"
                    >
                      11
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 12, 2020"
                    >
                      12
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 13, 2020"
                    >
                      13
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 14, 2020"
                    >
                      14
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 15, 2020"
                    >
                      15
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 16, 2020"
                    >
                      16
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 17, 2020"
                    >
                      17
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 18, 2020"
                    >
                      18
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 19, 2020"
                    >
                      19
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 20, 2020"
                    >
                      20
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 21, 2020"
                    >
                      21
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 22, 2020"
                    >
                      22
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 23, 2020"
                    >
                      23
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 24, 2020"
                    >
                      24
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 25, 2020"
                    >
                      25
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 26, 2020"
                    >
                      26
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 27, 2020"
                    >
                      27
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 28, 2020"
                    >
                      28
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 29, 2020"
                    >
                      29
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 30, 2020"
                    >
                      30
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="January 31, 2020"
                    >
                      31
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend react-calendar__month-view__days__day--neighboringMonth"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="February 1, 2020"
                    >
                      1
                    </abbr>
                  </button>
                  <button
                    className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend react-calendar__month-view__days__day--neighboringMonth"
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseOver={[Function]}
                    style={
                      Object {
                        "flexBasis": "14.285714285714286%",
                        "marginLeft": null,
                        "maxWidth": "14.285714285714286%",
                        "overflow": "hidden",
                      }
                    }
                    type="button"
                  >
                    <abbr
                      aria-label="February 2, 2020"
                    >
                      2
                    </abbr>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-reservation-row"
    >
      <div
        className="form-reservation-counter"
      >
        <p
          className="reservation-label"
        >
          Nombre de passager(s)
        </p>
        <div
          className="counter"
        >
          <button
            className="counter-button decrement-button"
            onClick={[Function]}
          >
            -
          </button>
          <h2
            className="counter-value"
          >
            0
          </h2>
          <button
            className="counter-button increment-button"
            onClick={[Function]}
          >
            +
          </button>
        </div>
      </div>
    </div>
    <input
      className="submitInput input-disabled"
      id="submitInput"
      type="submit"
      value="Valider"
    />
  </form>
</div>
`;
